### PR TITLE
Documented in_bulk behavior with nonexistent id_list items.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by claudep** [comment]
As usual with me, triple check language :-)

File: docs/ref/models/querysets.txt [hunk 1]
```diff
@@ -2284,7 +2284,10 @@ database query like ``count()`` would.
 
 Takes a list of field values (``id_list``) and the ``field_name`` for those
 values, and returns a dictionary mapping each value to an instance of the
-object with the given field value. If ``id_list`` isn't provided, all objects
+object with the given field value. No
+:exc:`django.core.exceptions.ObjectDoesNotExist` exceptions will ever be raised
+by ``in_bulk``, that is any ``id_list`` value not matching any instance will
```
Comment (jacobtylerwalls) [inline docs/ref/models/querysets.txt:2289]:
```suggestion
by ``in_bulk``; that is, any ``id_list`` value not matching any instance will
```

**Review by jacobtylerwalls — APPROVED** [review]
Looks good. In the commit message I would use "nonexistent" instead of "unexisting".

**Comment by claudep** [comment]
Thanks Jacob for the review!

**Review by carltongibson — APPROVED** [review]
Yep, I think it's fine. It's a tricky thing to express, and I think this is clear enough. Thanks @claudep

**Pull request merged**